plugins/web/active/Wapiti_Unauthenticated@OWTF-WVS-003.py

```diff
@@ -1,11 +1,15 @@
 from framework.dependency_management.dependency_resolver import ServiceLocator
-""" 
+"""
```

Comment:
Keep blank lines b/w imports and docstrings. Modify all files in `plugins/` with this fix.